egs2/mgb2/asr1/conf/tuning/train_asr_transformer3.yaml

```diff
@@ -0,0 +1,62 @@
+batch_type: numel
```

Comment:
For transformer as well. Literally, one config for each model architecture at most.